inventory-fusion/src/main/scala/com/normation/inventory/provisioning/fusion/FusionReportParsingExtension.scala

```diff
@@ -53,7 +53,9 @@ abstract class FusionReportParsingExtension extends PartialFunction[(Node,Invent
    */
   def optText(n:NodeSeq) = n.text match {
     case null | "" => None
-    case s => Some(s.trim().replaceAll("""[\p{Blank}]+"""," "))
+    case s => s.trim().replaceAll("""[\p{Blank}]+"""," ") match {
+      case ""   => None
+      case text => Some(text)
+    }
   }
```

Comment:
If it's exactly the same method, in the same project, we should dedup it. It really smell like bugs waiting to happen. 

For the code, it's ok. I'm not sure if it make more sense to have:
```
  def optText(n:NodeSeq) = n.text match {
    case null => None
    case s => s.trim().replaceAll("""[\p{Blank}]+"""," ") match {
      case ""   => None
      case text => Some(text)
    }
}
```
so that the blank case is at the same place, but it's really not important.